heartbeat/monitors/wrappers/summarizer/jobsummary/jobsummary.go

```diff
@@ -49,7 +49,11 @@ func NewJobSummary(attempt uint16, maxAttempts uint16, retryGroup string) *JobSu
 // BumpAttempt swaps the JobSummary object's pointer for a new job summary
 // that is a clone of the current one but with the Attempt field incremented.
 func (js *JobSummary) BumpAttempt() {
-	*js = *NewJobSummary(js.Attempt+1, js.MaxAttempts, js.RetryGroup)
+	newJs := *NewJobSummary(js.Attempt+1, js.MaxAttempts, js.RetryGroup)
+	newJs.Up = js.Up
+	newJs.Down = js.Down
+	newJs.Status = js.Status
```

Comment:
I don't understand why a new job would inherit the status of the old job, same for the `up` and `down` fields, those should be set during the execution of the new job. If this fixes the bug I don't understand why. Also, we need tests.

Does this code work if the first attempt is down, but the second is up? As I read it the answer would be no, but I may misunderstand